mojo/core/broker_castanets.h

```diff
@@ -152,14 +157,13 @@ class BrokerCastanets : public Channel::Delegate, public base::SyncDelegate {
                               bool write_lock = true);
 
   bool tcp_connection_ = false;
-  bool secure_connection_ = false;
 
   // Handle to the broker process, used for synchronous IPCs.
   PlatformHandle sync_channel_;
 
   // Channel endpoint connected to the inviter process. Recieved in the first
   // first message over |sync_channel_|.
-  PlatformChannelEndpoint inviter_endpoint_;
+  ConnectionParams inviter_endpoint_;
```

Comment:
It seems that we need to rename this member.
// Connection Params to connect to the inviter process. Recieved in the first
// first message over |sync_channel_|.
ConnectionParams inviter_connection_params_;

Reply:
Thank you. I'll update this to https://github.com/Samsung/Castanets/pull/262.